src/main/java/org/datadog/jmxfetch/Instance.java

```diff
@@ -186,6 +187,10 @@ public Instance(
             }
         }
 
+        if (initConfig != null) {
+            this.checkPrefix = (String) initConfig.get("check_prefix");
```

Comment:
I'd prefer naming this option `service_check_prefix` to reflect more closely what it does, since the prefix is only applied to the service check

Reply:
I was thinking it might be also used later for metrics if there is a need for that later. That's why I used `check_prefix` instead of `service_check_prefix`, WDYT?

Reply:
If we want to use such a prefix for metrics as well in the future, I think it'll be easier for backward compatibility to define a new config option (that would at that point be called `integration_prefix` for example) and apply it to both service checks and metrics; and still support `service_check_prefix` for service checks only.

Reply:
Sure, make sense. Changed to `service_check_prefix` instead.